docs/jupyter-lite.json

```diff
@@ -1,10 +1,6 @@
 {
   "jupyter-lite-schema-version": 0,
   "jupyter-config-data": {
-    "appName": "JupyterGIS App",
-    "disabledExtensions": [
-      "@jupyter/collaboration-extension",
-      "@jupyter/docprovider-extension"
-    ]
+    "appName": "JupyterGIS App"
```

Comment:
Since we're touching this file, let's take this opportunity to fix https://github.com/geojupyter/jupytergis/issues/369

```suggestion
```